src/app/modules/main/chat_section/view.nim

```diff
@@ -29,6 +29,7 @@ QtObject:
       chatsLoaded: bool
       communityMetrics: string # NOTE: later this should be replaced with QAbstractListModel-based model
       permissionsCheckOngoing: bool
+      isWaitingNewCommunityOwnerConfirmRequestToRejoin: bool
```

Comment:
Now seeing this variable and name a lot, even though it's already pretty long, it would sound better and be clearer as `isWaitingOnNewCommunityOwnerToConfirmRequestToRejoin`
Could you rename all of the names and events to that :see_no_evil:

Reply:
no problem